apps/amo/tests/test_celery.py

```diff
@@ -51,3 +53,21 @@ def test_handle_cache_miss_for_stats(self):
         self.cache.get.return_value = None  # cache miss
         fake_task.delay()
         assert not self.statsd.timing.called
+
+
+class TestTaskQueued(unittest.TestCase):
+    def setUp(self):
+        patch = mock.patch('amo.celery.statsd', autospec=True)
+        self.statsd = patch.start()
+        self.addCleanup(patch.stop)
+
+    def test_not_queued_outside_request_response_cycle(self):
+        fake_task.delay()
+        assert self.statsd.timing.called
+
+    def test_queued_inside_request_response_cycle(self):
+        request_started.send(sender=self)
+        fake_task.delay()
+        assert not self.statsd.timing.called
```

Comment:
what are these assertions meant for? They are already covered in the [task tracking tests](https://github.com/mozilla/olympia/blob/master/apps/amo/tests/test_celery.py#L47).